pkg/events/cloud/cloud_test.go

```diff
@@ -91,7 +100,7 @@ func TestCloudPubSub(t *testing.T) {
 		if a.So(e.Identifiers(), should.NotBeNil) && a.So(e.Identifiers(), should.HaveLength, 1) {
 			a.So(e.Identifiers()[0].GetApplicationIDs(), should.Resemble, &appID)
 		}
-	case <-time.After(time.Second):
+	case <-time.After(timeout):
```

Comment:
Create a context with timeout instead and just use `ctx.Done()` here.
Take a look at https://github.com/TheThingsNetwork/lorawan-stack/blob/d703d33381ce50b1966d44b003f7798e5bc7446a/pkg/util/test/test.go#L74-L165 as well